src/commands/database/mysql/user/update.tsx

```diff
@@ -0,0 +1,126 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import { Flags } from "@oclif/core";
+import { ReactNode } from "react";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import assertSuccess from "../../../../lib/apiutil/assert_success.js";
+import type { MittwaldAPIV2Client } from "@mittwald/api-client";
+type UpdateResult = void;
+type MyQSLUserUpdateData = Parameters<
+  MittwaldAPIV2Client["database"]["updateMysqlUser"]
+>[0]["data"];
+
+export default class Update extends ExecRenderBaseCommand<
+  typeof Update,
+  UpdateResult
+> {
+  static description = "Create a new mysql user";
+  static flags = {
+    ...processFlags,
+    "mysql-user-id": Flags.string({
+      required: true,
+      description: "MySQL User ID of the user to be updated",
+    }),
```

Comment:
Other commands that directly affect a specific resource use that resource ID as argument (comparison: all `delete` and `get` commands, and also other update commands like `mail address update`)